reference/library/src/morpheus-master/js/src/sakai.morpheus.responsive.menus.js

```diff
@@ -6,8 +6,14 @@ function toggleToolsNav(event){
   if (event) {
     event.preventDefault();
   }
+    
   $PBJQ('body').toggleClass('toolsNav--displayed');
-
+  if ($PBJQ('.Mrphs-portalBody').hasClass('toolsNav--displayed')) {
```

Comment:
Seems strange to have the search on line 10 for `body`, but the search on this line for `.Mrphs-portalBody` is there a case whtn you want the class to toggle but don't want the mask to in some cases?